shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/FirebaseAuthenticationApi.kt

```diff
@@ -2,11 +2,14 @@ import com.kcc.kmmhackathon.shared.entity.FirebaseAuthenticationResponse
 import io.ktor.client.HttpClient
 import io.ktor.client.features.*
 import io.ktor.client.request.*
+import io.ktor.client.statement.HttpStatement
+import io.ktor.client.statement.readText
 import io.ktor.client.features.json.JsonFeature
 import io.ktor.client.features.json.serializer.KotlinxSerializer
 import kotlinx.serialization.json.Json
 import io.ktor.http.*
 
+
 class FirebaseAuthenticationStore : AuthenticationStore {
 
  private val httpClient = HttpClient {
```

Comment:
I can see this was already like this, but can we indent it please

Reply:
Done